packages/bootstrap/lib/Wrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from "react"
+import { ThemeManager } from "@kaizen/design-tokens"
+import { themeSwitcher, getCurrentTheme } from "@cultureamp/theme-switcher"
+
+// Polyfill for :focus-visible pseudo-selector
+// See: https://github.com/WICG/focus-visible
+import "focus-visible"
+
+// Standard base stylesheet used across Culture Amp products
+// See: https://github.com/necolas/normalize.css/
+import "normalize.css"
+
+// Global styles
+import "./globals.module.scss"
+
+// Supported / active face declarations
+import "./fonts.module.scss"
+
+// Initialise the ThemeManager, this is a singleton so this should not be redeclared
+export const themeManager = new ThemeManager(getCurrentTheme())
```

Comment:
Should we split out the polyfills and global instances etc, from the React Wrapper?

If there's a haml page, or an Elm only app, without React, it'd be nice to have a simple file to import still.

(If you use the React wrapper, it could import that file for you)